portfolio/src/main/webapp/script.js

```diff
@@ -53,8 +54,8 @@ const handleClickProject = function() {
     });
 }
 
-const getComments = function() {
-    fetch('/data').then(response => response.json()).then((comments) => {
+const getComments = function(max) {
```

Comment:
When you do the

```javascript
const foo = function() {
};
```
you're essentially declaring a function variable, so this should end in a semicolon (as above).